Smooch.postman_collection.json

```diff
@@ -1328,7 +1328,7 @@
 					},
 					"response": [
 						{
-							"name": "Delete App User Example",
+							"name": "Delete App User Profile Example",
```

Comment:
Why is this updated?

Reply:
I just noticed you re-ordered the apis. Why?

Reply:
I had only added the generate media token and the get token api. The rest were changed by themselves. There is a thread on [#sunco-eng](https://zendesk.slack.com/archives/CJM6HKQUA/p1595363412301000) where I asked the exact same question.

Reply:
Postman has an annoying tendency to move entry-order around when editing/rebundling the collection; sorry bro.